liquidprompt

```diff
@@ -2730,7 +2758,7 @@ _lp_default_theme_prompt_template() {
         # add user, host and permissions colon
         PS1+="${LP_BRACKET_OPEN}${LP_USER}${LP_HOST}${LP_PERM}"
 
-        PS1+="${LP_PWD}${LP_DIRSTACK}${LP_BRACKET_CLOSE}${LP_SCLS}${LP_VENV}${LP_PROXY}"
+        PS1+="${LP_PWD}${LP_DIRSTACK}${LP_BRACKET_CLOSE}${LP_SCLS}${LP_AWS_PROFILE}${LP_VENV}${LP_PROXY}"
```

Comment:
Here you have `LP_AWS_PROFILE` before `LP_VENV`, but in `docs/theme/default.rst` you have it after. The order needs to be the same, as specified in that documentation:

> All of the available template sections are listed below. Their order is the default order if the user does not configure a different template.

I don't care which one comes first personally, just make them the same.

You also added it to the `liquid.ps1` file. You don't need to do that, see the comment in that file:
> This file is not updated with new template sections.

But if you do, make sure it is the same order as here.

Reply:
I've removed this from `liquid.ps1` and standardised the order elsewhere.